### PR TITLE
Added more logo variations to the Brand Kit

### PR DESCRIPTION
**Pull request by Captain-Wet-Beard**
Also moved a couple things and made things in the same category more uniform.

### CODE REVIEW

**Comment by sshiiden** [comment]
The flat design can be incorporated into the brand kit. [link to flat design](https://github.com/blazium-engine/blazium-assets/tree/c0a7265d15a161223f337ce1cad3f8857b8c1b05/Drag-n-Drop%20Assets/images_with_dimensions/editor/icons)

**Comment by Captain-Wet-Beard** [comment]
Good idea, I'll do that and further expand the pack with flat designs

**Comment by Captain-Wet-Beard** [comment]
Flat logos and also monochrome variants are added

**Review by Starkium — APPROVED** [review]

**Review by flare199 — APPROVED** [review]

**Review by Norrox — APPROVED** [review]

File: Blazium Engine - Brand Kit/Logo & Wordmark (horizontal)/PNG/Blazium Logo - White Outline.png
Comment (Starkium) [inline Blazium Engine - Brand Kit/Logo & Wordmark (horizontal)/PNG/Blazium Logo - White Outline.png:1]: what is the reason for the dimensions change

**Pull request merged**